### ISSUE TITLE
Publish new NuGet package for tool nservicebus.transport.azureservicebus.commandline

### ISSUE THREAD

**Issue by tmjnsn** [Particular/NServiceBus.Transport.AzureServiceBus#447]
**Is your feature request related to a problem? Please describe.**
When running our pipelines and using the latest version of the `asb-transport` tool (2.0.1), we run into the issue of a reference to an outdated .net core version (2.1). Our Azure DevOps pipelines no longer support this framework version.

```
The framework 'Microsoft.NETCore.App', version '2.1.0' was not found.
  - The following frameworks were found:
      3.1.21 at [/opt/hostedtoolcache/dotnet/shared/Microsoft.NETCore.App]
```

**Describe the solution you'd like**
Availability of a new NuGet package version of which the CommandLine project references .net core 3.1. This reference is already updated on the latest master branch, but was not yet published.

**Comment by danielmarbach** [comment]
Hi @tmjnsn 

Thanks for reporting this. Oddly enough, we have set the right TFMs

https://github.com/Particular/NServiceBus.Transport.AzureServiceBus/blob/release-2.0/src/CommandLine/NServiceBus.Transport.AzureServiceBus.CommandLine.csproj#L4

but it seems it got packaged into `tools\netcoreapp2.1\any`. We have to further investigate

@bording is the packaging tool something different there for tools? At first sight, I couldn't find anything suspicious yet.

**Comment by bording** [comment]
@danielmarbach We just changed the TFMs on the `release-2.0` branch as part of the work related to preparing for .NET 6 support. Before that change, it was only targeting `netcoreapp2.1`.

Releasing a new package with .NET Core 3.1 support and .NET 6 support is something we'll doing soon as part of that work.

**Comment by cquirosj** [comment]
@tmjnsn In the meantime, you can roll-forward (https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-environment-variables#dotnet_roll_forward) the tool on your build environment for it to run even when only dotnet 3 SDK is installed. I tested this by creating a docker container where only the SDK for 3.1 is installed. You can see the sample here https://gist.github.com/cquirosj/690e7426c255049b7a8fc95358bd912a

**Comment by tmjnsn** [comment]
We're currently executing this part of the pipeline on a dev machine with .net core 2.1. It works for now but it's not an ideal situation.

**Comment by boblangley** [comment]
Due to [an unrelated issue](https://github.com/Particular/NServiceBus.Transport.AzureServiceBus/issues/454), I have released [version 2.0.2](https://discuss.particular.net/t/nservicebus-transport-azureservicebus-2-0-2-critical-patch-release-available/2872) of the transport and command-line tool. It includes the TFM changes mentioned earlier, which changes the targets to .NET Core 3.1 and .NET 6.
